### PR TITLE
Create search by cuisine view.

### PR DESCRIPTION
**Pull request by SilverStrings024**
Created a view to search recipes or chefs by cuisine.

### CODE REVIEW

File: server/api/search.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                'location': chef.user.get_province_and_country,
+                'specialty': chef.favourite_recipe,
+                "about_me": chef.about_me
+            }
+    # All good, make the response and return
+    response_dict['status'] = 200
+    response = make_response(response_dict)
+    return response
+
+@search_views.route("/hi", methods=["GET"])
```
Comment (dproc96) [inline server/api/search.py]: I'm guessing this was a test route? You can remove it

File: server/app.py [hunk 2]
```diff
@@ -27,18 +27,8 @@
...
-def custom_user_loader_error(identity):
-    """
-    Identity is set to be the username
-    Make a more user friendly error
-    """
-    return  {
-        'status': 404,
-        "message": "User with username %s not found" % (identity)
-    }
+app.register_blueprint(search_views)
```
Comment (dproc96) [inline server/app.py]: I think you registered this blueprint twice

File: server/api/search.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    Return the proper info exampled in the mockup
+    or give back an error
+    """
+    search_by = request.args.get('search_by')
+    cuisine = request.args.get('cuisine')
+    result = []
+    response_dict = {}
+    # Make sure values were passed
+    if search_by != "recipe" and search_by != "chef":
+        return {
```
Comment (dproc96) [inline server/api/search.py]: Can you align your responses to this format `return jsonify({response}), status`?

**Review by dproc96 — COMMENTED** [review]
This looks like a good solution, had a couple of comments on implementation

File: server/api/search.py (hunk 1 above)
Comment (SilverStrings024) [reply server/api/search.py]: oops lol, I'll fix that now

File: server/app.py (hunk 2 above)
Comment (SilverStrings024) [reply server/app.py]: Okay, I'll remove that

File: server/api/search.py (hunk 3 above)
Comment (SilverStrings024) [reply server/api/search.py]: Sure, I'll do that now

**Pull request merged**